SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -238,15 +237,15 @@ void UGlobalStateManager::LinkExistingSingletonActor(const UClass* SingletonActo
 		SingletonActor->RemoteRole = ROLE_Authority;
 	}
 
-	// Add to entity registry
-	// This indirectly causes SetChannelActor to not create a new entity for this actor
-	NetDriver->GetEntityRegistry()->AddToRegistry(SingletonEntityId, SingletonActor);
-
-	Channel->SetChannelActor(SingletonActor);
-
 	// Since the entity already exists, we have to handle setting up the PackageMap properly for this Actor
 	NetDriver->PackageMap->ResolveEntityActor(SingletonActor, SingletonEntityId);
 
+	//// Add to entity registry
+	//// This indirectly causes SetChannelActor to not create a new entity for this actor
+	//NetDriver->GetEntityRegistry()->AddToRegistry(SingletonEntityId, SingletonActor);
```

Comment:
Problems with singletons?

Reply:
I think that code is safe to remove, just forgot to delete it.